src/helpers/message.ts

```diff
@@ -257,3 +258,37 @@ export function newOrderMessage(order: OrderDto, baseUrl: string): string {
   // Compose the final message
   return `${sellMsg}${priceMsg}${priceInverseMsg}${notYetActiveOrderMsg}${expirationMsg}${unknownTokenMsg}${fillOrderMsg}`
 }
+
+export interface SendMessageInput {
+  chatId: number | string
+  text: string
+  options?: TelegramBot.SendMessageOptions
+}
+
+export const MESSAGE_DELIMITER = '\n\n====================================================\n\n'
```

Comment:
critical issue: Maybe use one unicode from https://en.wikipedia.org/wiki/Box-drawing_character

```suggestion
export const MESSAGE_DELIMITER = '\n\n━━━━━━━━━━━━━━━━━━━━━━\n\n'
```